service/labelstore/service.go

```diff
@@ -56,12 +69,33 @@ func (s *service) Definition() agent_service.Definition {
 	}
 }
 
+func (s *service) Describe(m chan<- *prometheus.Desc) {
+	m <- s.totalIDs
+	m <- s.idsInRemoteWrapping
+}
+func (s *service) Collect(m chan<- prometheus.Metric) {
+	s.mut.Lock()
+	defer s.mut.Unlock()
+
+	m <- prometheus.MustNewConstMetric(s.totalIDs, prometheus.CounterValue, float64(len(s.labelsHashToGlobal)))
+	for name, rw := range s.mappings {
+		m <- prometheus.MustNewConstMetric(s.idsInRemoteWrapping, prometheus.CounterValue, float64(len(rw.globalToLocal)), name)
+	}
+}
+
 // Run starts a Service. Run must block until the provided
 // context is canceled. Returning an error should be treated
 // as a fatal error for the Service.
 func (s *service) Run(ctx context.Context, host agent_service.Host) error {
-	<-ctx.Done()
-	return nil
+	staleCheck := time.NewTicker(10 * time.Minute)
```

Comment:
We probably don't need to do this now, but do you think there may be a need to configure this in the future?

Reply:
Since staleness markers arent configurable and this was meant to be double the staleness marker length I dont think it will need to be configurable. Though that is famous last words.